### PR TITLE
Make ensure_pango_color support #rgba

### PR DESCRIPTION
**Pull request by psychon**
Reference: https://github.com/awesomeWM/awesome/pull/2129#discussion_r155397507

### CODE REVIEW

File: lib/gears/color.lua [hunk 1]
```diff
@@ -338,6 +338,11 @@ end
 -- @treturn string color if it is valid, else fallback.
```
Comment (actionless) [inline lib/gears/color.lua]: maybe mention in docs what it skip checking the certain "types" of colors?

Comment (psychon) [reply lib/gears/color.lua]: Sorry, what?

Comment (actionless) [reply lib/gears/color.lua]: my bad, i didn't noticed `%x` in the regexp so i thought it's only checking the length and if length seems like argb value it just returns it back

**Comment by actionless** [comment]
i think one week of conservation is enough for such a small change covered with the tests

**Pull request merged**